Gems/Atom/Feature/Common/Assets/ShaderResourceGroups/CoreLights/ViewSrg.azsli

```diff
@@ -83,13 +83,14 @@ partial ShaderResourceGroup ViewSrg
         float4x4 m_depthBiasMatrix;
         uint m_shadowmapArraySlice; // array slice who has shadowmap in the atlas.
         uint m_shadowFilterMethod; // Includes overall filter method in top 16 bits and pcf method in bottom 16 bits.
-        float m_boundaryScale;
         uint m_predictionSampleCount;
         uint m_filteringSampleCount;
-        float2 m_unprojectConstants;
+        float m_boundaryScale;
         float m_bias;
         float m_esmExponent;
-        float3 m_padding;
+        float m_padding0;
+        float2 m_unprojectConstants;
+        float2 m_padding1;
```

Comment:
We can use 1 padding in the end instead of 1 more in the middle.

Reply:
The extra padding is needed in order for the member to be 8 byte aligned. The error without m_padding0 is:

```
SPIR-V module not valid: Structure id 31 decorated as BufferBlock for variable in Uniform storage
class must follow standard storage buffer layout rules: member 8 at offset 92 is not aligned to 8
```

Reply:
yeah. without m_padding0, m_unprojectConstants would not align. It will get moved to a different register on the gpu.